tools/osbuilder/scripts/lib.sh

```diff
@@ -181,7 +181,8 @@ create_summary_file()
 	[ "$AGENT_INIT" = yes ] && agent="${init}"
 
 	local -r agentdir="${script_dir}/../../../"
-	local -r agent_version=$(cat ${agentdir}/VERSION)
+	local agent_version=$(cat ${agentdir}/VERSION 2> /dev/null)
+	[ -z "$agent_version" ] && agent_version="unknown"
```

Comment:
Okay, but then again, when the VERSION file wouldn't be present?

Reply:
See answer above.